examples/homography_estimation.py

```diff
@@ -268,21 +279,32 @@ def forward(self, img):
         return self.conv2(x)
 
 
-def run():
-    batch_size = 64
-    max_iterations = 50
-    step_size = 0.1
-    verbose = True
+def run(
+    batch_size: int = 64,
+    num_epochs: int = 999,
+    outer_lr: float = 1e-4,
+    max_iterations: int = 50,
+    step_size: float = 0.1,
+    autograd_mode: AutogradMode = AutogradMode.VMAP,
+):
+    logger.info(
+        "==============================================================="
+        "==========================="
+    )
+    logger.info(f"Batch Size: {batch_size}, " f"Autograd Mode: {autograd_mode}, ")
+
+    logger.info(
+        "---------------------------------------------------------------"
+        "---------------------------"
+    )
+    verbose = False
     imgH, imgW = 60, 80
     use_gpu = True
     viz_every = 10
-    disp_every = 10
     save_every = 100
-    num_epochs = 999
-    outer_lr = 1e-4
     use_cnn = True
 
-    log_dir = os.path.join(os.getcwd(), "examples/outputs/viz")
+    log_dir = os.path.join(os.getcwd(), "viz")
```

Comment:
Why remove the output subfolder `examples/outputs/`?

Reply:
We use `hydra` which has a default `outputs` directory.